### PR TITLE
T6 article comments

### PR DESCRIPTION
**Pull request by Jobeyobey**
- Complete t6
- Refactor some model code to destructure rows straight from response
- update 'nc-news-test.js' to 'app.test.js'
- **Removed db.end() from 'app.test.js' because db.end() in checkArticleExistsById() appears to cause db to be ended twice. Relying on just the db.end() in the util function appears to close the db effectively. Let me know if this is an issue, please!**

### CODE REVIEW

**Comment by Jobeyobey** [comment]
**Again, I forgot to add the endpoints to endpoints.json. I've added that in a new pull request here: https://github.com/Jobeyobey/be-nc-news/pull/10**

Also, I forgot to make a branch when handling t6. I tried to be smart by cherry-picking my commits to the new branch and then pushing, but have just realised this has ended up in my pull requests not working how I expected. I hope this is to blame for the 'add error testing...' commit exists in both pull requests.

File: __tests__/app.test.js [hunk 1]
```diff
@@ -9,10 +9,6 @@ beforeEach(() => {
     return seed(testData);
 });
 
-afterAll(() => {
```
Comment (LangaLee) [inline __tests__/app.test.js:12]: lets return this please

File: models/model-utils.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const db = require("../db/connection");
+
+afterAll(() => {
```
Comment (LangaLee) [inline models/model-utils.js:3]: lets remove this afterAll call as it will be called in our test file

**Review by LangaLee — COMMENTED** [review]
great work 🙂 lets remove the afterAll from the util file and return it to our test file. We should call it once in the test file to end all connections

**Pull request merged**